### DISCUSSION TITLE
最新的Armbian_bullseye_save_2023.06不支持RTL 8723DS网卡

### DISCUSSION THREAD

**Discussion by gracelsy** [ophub/amlogic-s9xxx-armbian#1515] [Allwinner]
最新的Armbian_bullseye_save_2023.06不支持RTL 8723DS网卡，能在下个版本添加下嘛

**Comment by gsjsjg** [comment]
https://github.com/lwfinger/rtl8723ds
这里支持，可自己编译加载模块
